object-construction-checker/src/main/java/org/checkerframework/checker/objectconstruction/MustCallInvokedChecker.java

```diff
@@ -160,6 +159,7 @@ private void checkPseudoAssignToOwning(Node node) {
     if (mustCallVal.isEmpty()) {
       return;
     }
+    increaseNumMustCall();
```

Comment:
Please add a comment explaining why we choose to increase the count here

Reply:
Actually, when I wanted to add a comment I noticed that here is not the right place to increment NumMustCall because checkPseudoAssignToOwning() is called not only in handleInvocation(), but also in handleTypeCast(). So, I updated the code to call incrementNumMustCall() only in handleInvocation() where the node is creating a new resource that is not skipped.
https://github.com/kelloggm/object-construction-checker/pull/249/commits/5260a7c49c6dc2c042e59816683cc968c422883c